.editorconfig

```diff
@@ -5,9 +5,9 @@ root = true
 [*]
```

Comment:
We should probably deploy this via https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot, are you interested to provide a patch to our modulesync_config repo? That would allow us to sync the `.editorconfig` to all of our repos.

Reply:
No, sorry but that's too much yak shaving for me. I will just revert it from this PR.